build.fsx

```diff
@@ -125,12 +127,18 @@ Target "RestoreDotNetCore" (fun _ ->
 
 Target "BuildDotNetCore" (fun _ ->
     !! "./**/project.json"
+    -- "./build/**"
     |> Fake.DotNetCli.Build id
 )
 
 Target "UnitTestsDotNetCore" (fun _ ->
-    !! "./Octokit.Next.Tests"
-    |> Fake.DotNetCli.Test id
+    [ "./Octokit.Tests"
+      "./Octokit.Tests.Conventions" ]
+    |> Seq.iter (fun d ->
+        Fake.DotNetCli.Test (fun p ->
```

Comment:
There seems to be a signature issue with this function:

```
Error:

        build.fsx(138,9): error FS0001: This expression was expected to have type
            unit
        but here has type
            'a -> unit
```

Throwing a `|> ignore` on the end seems to do the trick :trollface:

```fsharp
Fake.DotNetCli.Test (fun p ->
    { p with
        WorkingDir = d }) |> ignore
```